src/probotOnKube.ts

```diff
@@ -1,6 +1,7 @@
 import { Probot } from 'probot';
 import fs from 'fs';
-import k8s, { V1Secret } from '@kubernetes/client-node';
+import { V1Secret } from '@kubernetes/client-node';
+import * as k8s from '@kubernetes/client-node';
```

Comment:
why this change? does it provide some different functionality?

Reply:
'@kubernetes/client-node' didn't provide any default exports from what I could tell. My linter was giving warnings.

Reply:
ack, all good to me.
you would have to add signed commit to pass the DCO check.
use the `-s` argument while committing.